.github/workflows/github-actions-tests.yml

```diff
@@ -0,0 +1,18 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.12.x]
```

Comment:
What is the reason to go with 1.12 version? With SLE15 we start on 1.14 up to 1.17. I suspect Leap will be the same.

Reply:
Well, it was set for the project in [go.mod](https://github.com/uyuni-project/inter-server-sync/blob/tests/go.mod#L3) but I think we can safely update the version, as far as I can see.

Reply:
Got it, I'll update the version

Reply:
I rechecked and underlying SLE15SP3 has 1.17 already ( https://scc.suse.com/packages?name=SUSE%20Manager%20Server&version=4.2&arch=x86_64&query=go1.17&package_id=21579416 )
The same is true for SUMA 4.1 and SLE15SP2 ( https://scc.suse.com/packages?name=SUSE%20Manager%20Server&version=4.1&arch=x86_64&query=go1.17&package_id=21578693 )
So I see no reason to go with anything older then with 1.17.

Reply:
Just one note, the version in use should be the one supported by OBS to build the project for those OS's (I think is the same one supported in the OS).
In run-time we don't need go.